birdhouse/optional-components/secure-thredds/secure-access-magpie-permission.cfg

```diff
@@ -0,0 +1,33 @@
+permissions:
+  # this can be combined with 'all-public-access'
+  # it only ensures that it is set if not also employed
+  - service: thredds
```

Comment:
In the README, make clear to user this will make all of Thredds publicly readable, except those 2 secure folders.  Once enabled, to disable, user have to login to Magpie and perform manual changes, diactivating the component is not enough.  Similar to https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/optional-components#give-public-access-to-all-resources-for-testing-purposes

Reply:
On second though, do you even need this section?  Just let user decide what they want to expose themselves.  You just want to ensure those 2 folders are secured.

Reply:
Acutally, coming back on my previous comment. 
Yes, since the user can decide or not to add both the public-access + secure, no need to add the public one again here.